packages/dara-components/dara/components/graphs/components/causal_graph_viewer.py

```diff
@@ -52,7 +52,24 @@ class CausalGraphViewer(BaseGraphComponent):
     The causal graph can be edited by setting `editable=True`. The editor mode can be set to `EditorMode.DEFAULT`
     (default), `EditorMode.PAG` or `EditorMode.RESOLVER`. The `EditorMode.DEFAULT` mode assumes all edges
     are directed (the causal graph is a DAG). The `EditorMode.PAG` mode displays all edge types (beyond directed), while
-    `EditorMode.RESOLVER` allows users to confirm and accept edges.
+    `EditorMode.RESOLVER` allows users to confirm and accept edges. To change the editor mode, set the `editor_mode`
+    parameter.
+
+    ```python
+    from dara.components.graphs import CausalGraphViewer
+    from dara.components.graphs.definitions import EditorMode
```

Comment:
You can import them from the top level `from dara.components import EditorMode, CausalGraphViewer` makes it a bit easier for the example

Reply:
adapted across the docstrings - thanks!